js/registration.js

```diff
@@ -0,0 +1,51 @@
+function autocomplete(inp, arr) {
```

Comment:
I think it is a mistake to include this javascript. #6 asks for a static html css registration form. Lets keep things simple and expand on them with javascript in future PRs

Reply:
It's too late to remove the js from here. I did review already and need to do a final review and it would be merged.